man/systemd-journald.service.xml

```diff
@@ -287,7 +284,19 @@ systemd-tmpfiles --create --prefix /var/log/journal</programlisting>
         <filename>/var/log/journal</filename> is not available, or
         when <option>Storage=volatile</option> is set in the
         <citerefentry><refentrytitle>journald.conf</refentrytitle><manvolnum>5</manvolnum></citerefentry>
-        configuration file.</para></listitem>
+        configuration file.</para>
+
+	<para>When <filename>systemd-journald</filename> ceases writing to a journal file,
```

Comment:
hmm, sorry for noticing this only now, but there's something off with the indentation here... should be 2ch i figure, but this paragraph is one space further left than the one before

Reply:
Oops, it turns out that I need to copy most of .dir-locals because emacs does not inherit from the parent directory, but instead uses its own defaults for everything not specified, including tabs by default. I updated that commit that adds man/.dir-locals.el and removed any tabs that sneaked in.